org/lateralgm/components/ActionList.java

```diff
@@ -150,7 +146,7 @@ public void save()
 	 * @return The frame opened or <code>null</code> if no
 	 * frame was opened.
 	 */
-	public static MDIFrame openActionFrame(MDIFrame parent, Action a)
+	public MDIFrame openActionFrame(MDIFrame parent, Action a)
```

Comment:
It references FRAMES as `WeakReference<ActionFrame> fr = FRAMES.get(a);` which I am making non static.